nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -546,7 +546,7 @@ in
       [ nix
         pkgs.nix-info
       ]
-      ++ optional (config.programs.bash.enableCompletion && !versionAtLeast nixVersion "2.4pre") pkgs.nix-bash-completions;
+      ++ optional (config.programs.bash.enableCompletion) pkgs.nix-bash-completions;
```

Comment:
nit

```suggestion
      ++ optional config.programs.bash.enableCompletion pkgs.nix-bash-completions;
```

Reply:
I am going to merge this as-is considering that this is a really small nitpick (I also kinda prefer the way it currently is, I think it is easier to read).